src/main/resources/com/puppycrawl/tools/checkstyle/checks/coding/messages.properties

```diff
@@ -26,6 +26,7 @@ illegal.type=Declaring variables, return values or parameters of type ''{0}'' is
 inline.conditional.avoid=Avoid inline conditionals.
 instantiation.avoid=Instantiation of {0} should be avoided.
 magic.number=''{0}'' is a magic number.
+mismatch.package.directory=Package name is not same as directory.
```

Comment:
@MEZk 
Directory name is already in violation message, so I don't think adding it to the message is necessary.
I was contemplating package name to the message, but I am not sure if adds any value and will make the message extra long. @romani What you think?